manager/manager.go

```diff
@@ -332,6 +332,15 @@ func (m *Manager) ManageVolume(volumeID string) error {
 	stopCh := make(chan struct{})
 	m.managedVolumes[volumeID] = stopCh
 
+	// Create a context that will be cancelled when the stopCh is closed
+	ctx, cancel := context.WithCancel(context.Background())
+	go func() {
+		select {
+		case <-stopCh:
+			cancel()
+		}
+	}()
```

Comment:
question: could we use a context instead of `stopCh`?

```go
	ctx, cancel := context.WithCancel(context.Background())
	m.managedVolumes[volumeID] = ctx.Done
```

Reply:
Never mind, I just read that `ctx.Done` can't be closed since it is "managed" by the Context package 😇